lib/registries/addon/overview/-components/diff-manager/template.hbs

```diff
@@ -1,5 +1,7 @@
-{{did-update this.recalculateDiff}}
+{{did-update this.recalculateDiff @headRevisionId}}
```

Comment:
For the render modifiers to listen to an argument/parameter/property, we need to pass it as a param to the modifier.